benchmark/README.md

```diff
@@ -1,417 +1,176 @@
-# Node.js core benchmark
+# Node.js Core Benchmarks
 
-This folder contains benchmarks to measure the performance of the Node.js APIs.
+This folder contains code and data used to measure performance
+of different Node.js implementations and different ways of
+writing JavaScript run by the built-in JavaScript engine.
+
+For a detailed guide on how to write and run benchmarks in this
+directory, see [the guide on benchmarks](../doc/guides/writing-and-running-benchmarks.md).
 
 ## Table of Content
 
-* [Prerequisites](#prerequisites)
-* [Running benchmarks](#running-benchmarks)
- * [Running individual benchmarks](#running-individual-benchmarks)
- * [Running all benchmarks](#running-all-benchmarks)
- * [Comparing node versions](#comparing-node-versions)
- * [Comparing parameters](#comparing-parameters)
-* [Creating a benchmark](#creating-a-benchmark)
+* [Benchmark directories](#benchmark-directories)
+* [Common API](#common-api)
+
+## Benchmark Directories
+
+### arrays
```

Comment:
Having these as a table may be more concise. Separate headers doesn't seem necessary